cores/arduino/USBCore.cpp

```diff
@@ -175,22 +184,25 @@ size_t EPBuffer<L>::sendSpace()
 template<size_t L>
 void EPBuffer<L>::flush()
 {
+    usb_disable_interrupts();
     // Don't flush an empty buffer
     if (this->len() == 0) {
         return;
+        usb_enable_interrupts();
```

Comment:
Will this actually get called after the return statement just above it?

Reply:
Oof, nice catch. Will fix.

Reply:
Thanks! Updated.